mlir/utils/jenkins/Dockerfile.migraphx-ci

```diff
@@ -1,5 +1,5 @@
-FROM rocm/mlir:rocm5.3-latest
-ARG ROCM_PATH=/opt/rocm-5.3.2
+FROM rocm/mlir:rocm5.4.2-latest
```

Comment:
I think these `FROM` lines are usually spelled `mlir:rocm5.4-latest`

Reply:
Judging from past commits, it looks like you are correct.  Updated.